Mathlib/RingTheory/Polynomial/Cyclotomic/Basic.lean

```diff
@@ -47,11 +47,8 @@ results hold if there is a primitive `n`-th root of unity in `R`. In particular,
 not the standard one unless there is a primitive `n`th root of unity in `R`. For example,
 `cyclotomic' 3 ℤ = 1`, since there are no primitive cube roots of unity in `ℤ`. The main example is
 `R = ℂ`, we decided to work in general since the difficulties are essentially the same.
-To get the standard cyclotomic polynomials, we use `int_coeff_of_cycl`, with `R = ℂ`, to get a
-polynomial with integer coefficients and then we map it to `R[X]`, for any ring `R`.
-To prove `cyclotomic.irreducible`, the irreducibility of `cyclotomic n ℤ`, we show in
-`cyclotomic_eq_minpoly` that `cyclotomic n ℤ` is the minimal polynomial of any `n`-th primitive root
-of unity `μ : K`, where `K` is a field of characteristic `0`.
+To get the standard cyclotomic polynomials, we use `unique_int_coeff_of_cycl`, with `R = ℂ`,
```

Comment:
It looks like some part of the docstring got lost...

Reply:
This part was split off in leanprover-community/mathlib#19144.

Reply:
aah, ok